content/applications/essentials/activities.rst

```diff
@@ -2,237 +2,345 @@
+  reveal a drop-down menu. Select the desired time-frame option from the list. The options are
+  :guilabel:`Days`, :guilabel:`Weeks`, or :guilabel:`Months`.
+- :guilabel:`Workspace`: using the drop-down menu, select the specific :ref:`Workspace
+  <documents/workspaces>` the document is being uploaded to.
+- :guilabel:`Tags`: select any desired tags from the drop-down menu. The available tags displayed
+  are based on the tags configured for the selected :guilabel:`Workspace`.
+- :guilabel:`Message`: enter a message to clarify the document request in this field.
 
-.. note::
-   The actions available to select on an activity type vary, depending on the applications currently
-   installed in the database.
+When all the fields are completed. click :guilabel:`Request` to send the document request.
 
-Default user
-~~~~~~~~~~~~
+.. image:: activities/request-doc.png
+   :align: center
+   :alt: The Request a file form, with all fields filled out to request a contract.
 
-To automatically assign this activity to a specific user when this activity type is scheduled,
-choose a name from the :guilabel:`Default User` drop-down menu. If this field is left blank, the
-activity is assigned to the user who creates the activity.
+.. _activities/types:
 
-Default summary
-~~~~~~~~~~~~~~~
+Activity types
+==============
 
-To include notes whenever this activity type is created, enter them into the :guilabel:`Default
-Summary` field.
+To view the currently configured types of activities in the database, navigate to
+:menuselection:`Settings app --> Discuss --> Activities --> Activity Types`.
 
-.. note::
-   The information in the :guilabel:`Default User` and :guilabel:`Default Summary` fields are
-   included when an activity is created. However, they can be altered before the activity is
-   scheduled or saved.
+.. image:: activities/settings-activities-types.png
+   :align: center
+   :alt: Activity Types button in the Settings application under the Discuss section.
 
-Next activity
--------------
+Doing so reveals the :guilabel:`Activity Types` page, where the existing activity types are found.
 
-To automatically suggest, or trigger, a new activity after an activity has been marked complete, the
-:guilabel:`Chaining Type` must be set.
+.. image:: activities/activity-list.png
+   :align: center
+   :alt: The list of activity types already configured and available.
 
-Suggest next activity
-~~~~~~~~~~~~~~~~~~~~~
+Edit activity types
+-------------------
 
-In the :guilabel:`Chaining Type` field, select :guilabel:`Suggest Next Activity`. Upon doing so, the
-field underneath changes to: :guilabel:`Suggest`. Click the :guilabel:`Suggest` field drop-down menu
-to select any activities to recommend as follow-up tasks to this activity type.
+To edit an existing :ref:`activity type <activities/types>`, click on the activity type, and the
+:guilabel:`Activity Type` form loads.
```

Comment:
```suggestion
activity type form loads.
```
technically, this isn't what the page is titled on the UI...so a GUI is not applicable in this case, in my opinion

Reply:
I was torn, because the name doesn't appear when doing this specific flow, BUT the form is identical/is the same as the "acivity type" form- which does say those words when mking an activity in another way. Since it was the same fom, I used the GUI-BUT I completely understand your logic. I admit, i do not understand why they didn't have the title for the form appear here when it is clearly the same form.